src/picongpu/include/traits/attribute/GetCharge.hpp

```diff
@@ -48,40 +48,40 @@ struct LoadBoundElectrons
     /** Functor implementation
      *
      * \tparam T_Particle particle type
-     * \param singlyChargedResult charge resulting from multiplying a single
+     * \param weighting the particle's weighting
      * electron charge (positive OR negative) by the macro particle weighting
      * \param particle particle reference
      */
     template<typename T_Particle>
-    HDINLINE float_X operator()(const float_X singlyChargedResult, const T_Particle& particle)
+    HDINLINE float_X operator()(const float_X weighting, const T_Particle& particle)
     {
         const float_X protonNumber = GetAtomicNumbers<T_Particle>::type::numberOfProtons;
 
-        return singlyChargedResult * (protonNumber - particle[boundElectrons_]);
+        return
+            ELECTRON_CHARGE *
+            ( particle[boundElectrons_] - protonNumber ) *
```

Comment:
OPTIONAL:
Here a comment on `( particle[boundElectrons_] - protonNumber )`being a negative number and thus `ELECTRON_CHARGE * ( particle[boundElectrons_] - protonNumber )` being a positive value would be helpful.